SimPEG/simulation.py

```diff
@@ -128,11 +158,12 @@ def counter(self, value):
 
     @property
     def sensitivity_path(self):
-        """Path to store the sensitivity.
+        """Path to directory where sensitivity file is stored.
```

Comment:
Just a quick double-check: Is it the path to the directory? or is it the path to the file?

Reply:
The comment below specifies that it is the file

Reply:
In most simulations, the `sensitivity_path` should be a directory. In the only case I'm aware this is not the case is with the new Choclo-based simulations, where the `sensitivity_path` should be a file where the `numpy.memmap` will be stored. I think we can use "directory" here and then update these methods for the mag and gravity simulations in #1321.